src/components/Navbar/Cart.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import '../../App.css';
+import { CartItems } from './CartItems';
+import cart from './icons/cart.svg';
+
+class Cart extends React.Component {
+    state = { dropdownVisible: false } 
```

Comment:
I think with react 17 you can use Hooks to use state

Reply:
Ok, changed to hook